index.html

```diff
@@ -231,6 +231,14 @@ <h3>Latest news</h3>
         </div>
         <div class="js-home-carousel home-carousel">
           <!-- News box Start -->
+        <div class="news-box">
+        <div class="news-box-title">
+            <span>15/03/2021</span>
+        </div>
+        <div class="news-box-content">
+            <p>Uyuni is participating in <a href="https://developers.google.com/season-of-docs" target="_blank">Google Season of Docs</a>. If you're a technical writer who would like to contribute to the Uyuni documentation, stay tuned for more information.</p>
```

Comment:
Shouldn't we point to a place where we have ideas for the GSoD? Or tell people how to propose ideas? Just as we did for the GSoC? @paususe?

Reply:
I'm not 100% certain what the status is, I need Pau to weigh in, because he's organising this. I just wrote the proposals.